test/lib/cloningTest.js

```diff
@@ -12,30 +12,30 @@ describe("Fetching redirected resources", () => {
   function fakeResponse(pathName) {
     fake
       .get(pathName)
-      .reply(200, {a: 1, b: 2});
+      .reply(200, { a: 1, b: 2 });
```

Comment:
another unnecessary rule IMO.

Reply:
its' part of the standard to ensure that the same spacing is used everywhere